src/components/dashboard/Overview/Overview.tsx

```diff
@@ -139,7 +141,9 @@ const Overview = (): ReactElement => {
                     <Typography color="border.main" variant="body2">
                       Tokens
                     </Typography>
-                    <StyledText fontSize="lg">{tokenCount || <ValueSkeleton />}</StyledText>
+                    <StyledText fontSize="lg">
+                      {tokenCount || hasHiddenAllTokens ? tokenCount : <ValueSkeleton />}
```

Comment:
I would attach this to 
```ts
const { balances, loading } = useVisibleBalances()
{/* ... */}
<StyledText fontSize="lg">
  {loading ? <ValueSkeleton /> : tokenCount }
</StyledText>
```

Reply:
When the `loading` flag changes, it will initially show `0` tokens before the `tokenCount`. I've changed the logic to not allow for the hiding of native token now. I think it's a cleaner solution.